diff -Naur vokoscreen-2.0.0.orig/main.cpp vokoscreen-2.0.0/main.cpp
--- vokoscreen-2.0.0.orig/main.cpp	2014-09-21 18:19:03.728207601 +0200
+++ vokoscreen-2.0.0/main.cpp	2014-09-21 18:39:30.346209759 +0200
@@ -12,7 +12,8 @@
  *
  * You should have received a copy of the GNU General Public License
  * along with this program; if not, write to the Free Software
- * Foundation, Inc., 59 Temple Place - Suite 330, Boston, MA 02111-1307, USA.
+ * Free Software Foundation, Inc., 51 Franklin Street, Fifth Floor,
+ * Boston, MA  02110-1301 USA
  */
 
 #include "screencast.h"
diff -Naur vokoscreen-2.0.0.orig/regionselection.cpp vokoscreen-2.0.0/regionselection.cpp
--- vokoscreen-2.0.0.orig/regionselection.cpp	2014-03-19 08:46:47.000000000 +0100
+++ vokoscreen-2.0.0/regionselection.cpp	2014-09-21 18:37:59.185070848 +0200
@@ -12,7 +12,8 @@
  *
  * You should have received a copy of the GNU General Public License
  * along with this program; if not, write to the Free Software
- * Foundation, Inc., 59 Temple Place - Suite 330, Boston, MA 02111-1307, USA.
+ * Free Software Foundation, Inc., 51 Franklin Street, Fifth Floor,
+ * Boston, MA  02110-1301 USA 
  */
 
 #include "regionselection.h" 
diff -Naur vokoscreen-2.0.0.orig/regionselection.h vokoscreen-2.0.0/regionselection.h
--- vokoscreen-2.0.0.orig/regionselection.h	2014-03-19 08:46:47.000000000 +0100
+++ vokoscreen-2.0.0/regionselection.h	2014-09-21 18:38:59.986163490 +0200
@@ -12,7 +12,8 @@
  *
  * You should have received a copy of the GNU General Public License
  * along with this program; if not, write to the Free Software
- * Foundation, Inc., 59 Temple Place - Suite 330, Boston, MA 02111-1307, USA.
+ * Free Software Foundation, Inc., 51 Franklin Street, Fifth Floor,
+ * Boston, MA  02110-1301 USA 
  */
 
 #ifndef regionselection_H 
diff -Naur vokoscreen-2.0.0.orig/screencast.cpp vokoscreen-2.0.0/screencast.cpp
--- vokoscreen-2.0.0.orig/screencast.cpp	2014-03-19 08:46:47.000000000 +0100
+++ vokoscreen-2.0.0/screencast.cpp	2014-09-21 18:40:21.660287949 +0200
@@ -12,8 +12,10 @@
  *
  * You should have received a copy of the GNU General Public License
  * along with this program; if not, write to the Free Software
- * Foundation, Inc., 59 Temple Place - Suite 330, Boston, MA 02111-1307, USA.
+ * Free Software Foundation, Inc., 51 Franklin Street, Fifth Floor,
+ * Boston, MA  02110-1301 USA 
  */
+
 #include "screencast.h"  
 
 using namespace std;
diff -Naur vokoscreen-2.0.0.orig/screencast.h vokoscreen-2.0.0/screencast.h
--- vokoscreen-2.0.0.orig/screencast.h	2014-03-19 08:46:47.000000000 +0100
+++ vokoscreen-2.0.0/screencast.h	2014-09-21 18:37:16.953006540 +0200
@@ -12,7 +12,8 @@
  *
  * You should have received a copy of the GNU General Public License
  * along with this program; if not, write to the Free Software
- * Foundation, Inc., 59 Temple Place - Suite 330, Boston, MA 02111-1307, USA.
+ * Free Software Foundation, Inc., 51 Franklin Street, Fifth Floor,
+ * Boston, MA  02110-1301 USA
  */
 
 #ifndef screencast_H 
